### ISSUE TITLE
Create: Game Data

### ISSUE THREAD

**Issue by hentai-chan** [cpp-gamedev/pkmn#4]
It would be nice if we didn't have to hardcode anything into the game so let's use the Pokemon API:

```
https://pokeapi.co/api/v2/pokemon/bulbasaur
```

This response only contains the name and an another API endpoint about the move known to this Pokemon at all times, for example

```
https://pokeapi.co/api/v2/move/14/
```

For the sprites we'd have to use a different API (the `id` can be found in the previous response):

```
https://pokeres.bastionbot.org/images/pokemon/1.png
```

After that comes the interesting part: generating the ASCII images from the original sprites. I'll will prototype something in python first. If possible, I'd like to rewrite this in C++ later. 

---

In summary, for one match alone we'd need to make as many as `2*(1+4+1)=12` requests. I think it would be best if we have some default game data for testing purposes, and then another procedure to fetch all data before the game starts.

--- 

### Task List

- [x] implement a function that collects game data from API calls
- [x] implement a function that creates ASCII images from sprites
- [x] define `requirements.txt` and add python-specific entries to `.gitignore`
- [x] explain basic usage in `README.md`

**Comment by hentai-chan** [comment]
After a change of heart, I decided to add all moves to the resulting JSON file. This means that this script (`gen_data.py`) makes between 1 and 70ish API calls per Pokemon, the bulk work of which comes from all the `move` endpoints because some moves have rather complicated in-game logic. Whether this is the case can only be reasoned with after the calls have been made because the base API initially only provides the `slug` and `endpoint` of each move, so I might as well hand over to @NikolaJelic the task to pick some good moves from this set.

```cli
# basic usage
python gen_data.py --help
```
